core/remoteviewserver.cpp

```diff
@@ -228,5 +228,6 @@ void RemoteViewServer::clientConnectedChanged(bool connected)
 void RemoteViewServer::requestUpdateTimeout()
 {
     emit requestUpdate();
+    emit frameRequested();
```

Comment:
Why the extra signal, isn't requestUpdate() emitted at exactly the same time? Testing that would even be better, as that's what will be what the using code will follow.